### ISSUE TITLE
Ansible must display playbook's filename and line number in errors

### ISSUE THREAD

**Issue by vhosakot** [ansible/ansible#14816]
##### Issue Type:
- Bug Report
##### Ansible Version:

```
ansible 1.9.4
  configured module search path = None
```
##### Ansible Configuration:

ansible.cfg has:

```
[defaults]
remote_tmp = /tmp
log_path = /var/log/kolla/ansible.log
```

No ANSIBLE_\* environment variables set.
##### Environment:

Ubuntu 14.04
##### Summary:

When I run  `ansible-playbook <playbook filename>`  and if there is an error, Ansible does not display the playbook's filename and line number in errors. This makes debugging **very** tough.
##### Steps To Reproduce:

Create the Ansible directory structure according to Ansible Best Practices mentioned in http://docs.ansible.com/ansible/playbooks_best_practices.html#directory-layout. Make an error/mistake in one of the playbooks and run the master playbook.
##### Expected Results:

```
Error when running /home/mypath/site.yml, line 23
Error when running /SomePath/roles/SomeModule/tasks/main.yml, line 49
```
##### Actual Results:

Error below run with -vvv

```
PLAY [ceph-mon;glance-api;glance-registry;rabbitmq] *************************** 

GATHERING FACTS *************************************************************** 
<localhost> REMOTE_MODULE setup
<localhost> EXEC ['/bin/sh', '-c', 'mkdir -p $HOME/.ansible/tmp/ansible-tmp-1457128980.63-76766687891340 && echo $HOME/.ansible/tmp/ansible-tmp-1457128980.63-76766687891340']
<localhost> PUT /tmp/tmpcArDhg TO /home/kolla/.ansible/tmp/ansible-tmp-1457128980.63-76766687891340/setup
<localhost> EXEC ['/bin/sh', '-c', u'LANG=en_US.UTF-8 LC_CTYPE=en_US.UTF-8 /usr/bin/python /home/kolla/.ansible/tmp/ansible-tmp-1457128980.63-76766687891340/setup; rm -rf /home/kolla/.ansible/tmp/ansible-tmp-1457128980.63-76766687891340/ >/dev/null 2>&1']
failed: [localhost] => {"cmd": "/bin/lsblk -ln --output UUID /dev/mapper/kolla--ubuntu--vg-root", "failed": true, "rc": 257}
msg: Traceback (most recent call last):
  File "/home/kolla/.ansible/tmp/ansible-tmp-1457128980.63-76766687891340/setup", line 1684, in run_command
    rfd, wfd, efd = select.select(rpipes, [], rpipes, 1)
  File "/home/kolla/.ansible/tmp/ansible-tmp-1457128980.63-76766687891340/setup", line 1823, in _handle_timeout
    raise TimeoutError(error_message)
TimeoutError: Timer expired
```

The filename and line numbers displayed in the errors above do not point to the playbook(s) written by the user.

**Comment by bcoca** [comment]
in this case the line number is 'None' as the fact gathering is built in by default

**Comment by vhosakot** [comment]
How will the user know what file and line number caused the errors ?

Shouldn't Ansible display default's playbook filename and line number ?

Below is an example.

```
Error when running /SomePath/roles/common/defaults/main.yml, line 29
```

**Comment by bcoca** [comment]
in versions 2.0  and above you do get this information, except for this task in which you should only get filename as the task is not defined in any line number.

**Comment by bcoca** [comment]
closing this as 'fixed in 2.0'
